packages/agreement-process/src/model/domain/models.ts

```diff
@@ -48,3 +52,52 @@ export const CompactEService = z.object({
   name: z.string(),
 });
 export type CompactEService = z.infer<typeof CompactEService>;
+
+export type AgreementContractPDFPayload = {
+  todayDate: string;
+  todayTime: string;
+  agreementId: AgreementId;
+  submitterId: UserId;
+  submissionDate: string;
+  submissionTime: string;
+  activatorId: UserId;
+  activationDate: string;
+  activationTime: string;
+  eserviceId: EServiceId;
+  eserviceName: string;
+  descriptorId: DescriptorId;
+  descriptorVersion: string;
+  producerName: string;
+  producerIpaCode: string | undefined;
+  consumerName: string;
+  consumerIpaCode: string | undefined;
+  certifiedAttributes: Array<{
+    assignmentDate: string;
+    assignmentTime: string;
+    attributeName: string;
+    attributeId: string;
+  }>;
+  declaredAttributes: Array<{
+    assignmentDate: string;
+    assignmentTime: string;
+    attributeName: string;
+    attributeId: string;
+  }>;
+  verifiedAttributes: Array<{
+    assignmentDate: string;
+    assignmentTime: string;
+    attributeName: string;
+    attributeId: string;
+    expirationDate: string | undefined;
+  }>;
+  producerDelegationId: string | undefined;
+  producerDelegatorName: string | undefined;
+  producerDelegatorIpaCode: string | undefined;
+  producerDelegateName: string | undefined;
+  producerDelegateIpaCode: string | undefined;
+  consumerDelegationId: string | undefined;
+  consumerDelegatorName: string | undefined;
+  consumerDelegatorIpaCode: string | undefined;
+  consumerDelegateName: string | undefined;
+  consumerDelegateIpaCode: string | undefined;
```

Comment:
By removing question marks here I enforce that these fields shall always be explicitly set to some value, either string or undefined. An example of what I proposed here: https://pagopa.atlassian.net/browse/PIN-5598